copier/user_data.py

```diff
@@ -193,15 +196,16 @@ class Question:
     placeholder: str = ""
     secret: bool = False
     type: str = ""
+    validator: str = ""
```

Comment:
This new property deserves a space in the class docstring! (which is rendered in the docs BTW)